main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -151,7 +151,7 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
 
   def UsesOrImports: Rule1[Seq[ParsedAst.UseOrImport]] = rule {
     // It is important for documentation comments that whitespace is not consumed if no uses are present
-    (optWS ~ oneOrMore((Use | Import) ~ optWS ~ ";").separatedBy(optWS)) | push(Seq.empty)
+    (optWS ~ oneOrMore((Use | Import) ~ optWS ~ optional(";")).separatedBy(optWS)) | push(Seq.empty)
```

Comment:
This is not completely correct because it would allow two uses next to eachother without whitespace, e.g. `use FoouseBar`.

Reply:
We want something like either there is optWS and a semicolon or there is hard WS.

Reply:
Good point. Fixed, I think.

Reply:
`oneOrMore((Use | Import) ~ optWS ~ optional(";")).separatedBy(optWS)` this sequence could also end on optWS and thus eat the doc comments

Reply:
@JonathanStarup I think my change and your comment crossed in the aether. Does what I've just pushed address your concern?

Reply:
The final version looks good 👍